src/main/resources/en/aviation.yml

```diff
@@ -1,7 +1,7 @@
 en:
   faker:
     aviation:
-      aircraft: ["An-2", "An-3", "An-8", "An-10", "An-12", "An-24", "An-26", "An-32", "An-38", "An-70", "An-72", "An-74TK", "An-124", "An-225", "Tu-2", "Tu-4", "Tu-16", "Tu-95", "Tu-104", "Tu-114", "Tu-128", "Tu-134", "Tu-144", "Tu-154", "Tu-154M", "Tu-160", "Tu-204", "Tu-214", "Tu-334", "Il-2", "Il-4", "DB-3", "Il-28","Il-38", "Il-76", "Il-86", "Il-96", "Il-112", "Il-114", "Li-2", "Yak-1", "Yak-3", "Yak-9", "Yak-12", "Yak-18A", "Yak-18T", "Yak-52", "Yak-55", "Yak-36", "Yak-38", "Yak-130", "Yak-141", "Yak-152", "Su-2", "Su-7", "Su-7B", "Su-15", "Su-17", "Su-20", "Su-22", "Su-24", "Su-25", "Su-26", "Su-27", "Su-29", "Su-30", "Su-30MK", "Su-30MKI", "Su-32FN", "Su-34", "Su-35", "Su-37", "Su-57", "MiG-1", "MiG-3", "MiG-9", "MiG-15", "MiG-17", "MiG-19", "MiG-21", "MiG-23", "MiG-25", "MiG-29", "MiG-35", "LaGG-3", "La-5", "La-5FN", "La-7", "La-9", "Be-12", "Be-200", "VVA-14", "Cessna-172", "Cessna-182", "Cessna-152", "Cessna-120", "Cessna-140", "Cessna-208", "Mi-8", "Mi-24", "Mi-35", "Mi-17", "Mi-28", "Ka-25", "Ka-27", "Ka-50", "Ka-52", "Po-2", "U-2", "SSJ", "MS-21", "Airbus 300", "Airbus 319", "Airbus 320", "Airbus 321", "Airbus 330", "Airbus 340", "Airbus 350", "Airbus 380", "Boeing 707", "Boeing 727", "Boeing 737", "Boeing 747", "Boeing 757", "Boeing 767", "Boeing 777", "Boeing 787", "C-47", "C-130", "de Havilland Canada DHC-2 Beaver", "de Havilland Canada DHC-6 Twin Otter", "de Havilland Comet", "de Havilland Dragon", "de Havilland Sea Venom", "Piper PA-28 Cherokee", "Piper J-3 Cub", "L-410", "L-29", "L-39", "Zenith STOL CH 701", "Zenair CH 640", "Zmaj Fizir FN", "Zlín Z-50", "Beechcraft Baron", "Beechcraft Queen Air", "Saab 35 Draken", "Saab JAS 39 Gripen", "Dassault Rafale", "Dassault Mirage F1", "Dassault Mirage III", "T-38", "F-4", "F-104", "F-5", "F-105", "F-101", "F-111", "F-15", "F-16", "F-18", "F-117", "F-22", "F-35", "PC-21", "P-39", "P-40", "P-63", "A-5", "A-7", "A-9", "B-25", "B-12", "B-2", "B-52", "Catalina", "Ju-87", "Ju-88", "He-111", "Fw-190", "Bf-109", "Bf-110", "Spitfire", "Hurricane", "Harrier", "Tornado"]
+      aircraft: [Airbus A220-100, Airbus A220-300, Airbus A318, Airbus A319, Airbus A320, Airbus A320neo, Airbus A321, Airbus A321neo, Airbus A330-200, Airbus A330-200F, Airbus A330-300, Airbus A330-800neo, Airbus A330-900neo, Airbus A340, Airbus A340-300, Airbus A340-500, Airbus A340-600, Airbus A350, Airbus A350-900, Airbus A350-1000, Airbus A380, Airbus A380-800, Boeing 707, Boeing 717, Boeing 727, Boeing 737, Boeing 747, Boeing 757, Boeing 767, Boeing 777, Boeing 787, Boeing 747-8, Boeing 737 MAX, Antonov An-12, Antonov An-24, Antonov An-26, Antonov An-28, Antonov An-30, Antonov An-32, Antonov An-72, Antonov An-124 Ruslan, Antonov An-140, Antonov An-148, Antonov An-158, Antonov An-225 Mriya, McDonnell Douglas MD11, McDonnell Douglas MD80, McDonnell Douglas MD81, McDonnell Douglas MD82, McDonnell Douglas MD83, McDonnell Douglas MD87, McDonnell Douglas MD88, McDonnell Douglas MD90]
```

Comment:
Is there a reason to reduce this instead of just adding new ones?

Reply:
1) there are helicopters and airplanes, maybe it makes sense to have helicopters and planes lists
2) it contains mainly warplanes from WWII, which is, for me, less appropriate for most cases than civil planes used by airlines

Reply:
For my two cents maybe create a new category/provider for ww2 (or military) planes and helicopters?

If it makes sense (I'm no aeronautics expert) you could separate helicopters in to military and civilian or commercial as well.

Reply:
But I'm not sure what to do with the `aircraft()` method in Aviation, we need to preserve backward compatibility and at the same we're removing support of aircrafts and introduce airplanes and helicopters instead

Reply:
I'd have to dig around, I've only been working with the project for a month or two. I wonder if there's a way to resolve from within a hierarchy?

```yaml
aircraft:
   airplanes: plane1, plane2
   helicopters: helo1, helo2
```

So `aircraft()` would resolve a random value from either list.